andela/Titan-Devs
# #163518699 Calculate Time Length to Read Article

#### What does this PR do?
- It attaches a readTime attribute on the created article. 
#### Description of Task to be completed?
- Add readTime attribute on an article. 
#### How should this be manually tested?
- After cloning the repository, check out `ft-time-lenght-to-read-article-163518699` branch
- Make sure to read `README.md` file to get the setup instructions
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Create an article using `POST /api/v1/articles` endpoint
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
- the request body should be as mentioned in the `README.md` file under `Endpoints:` ==> `Create Article` subtitle
After the successful article creation, you should get back the article information. With the successful message, and readTime attribute.

#### What are the relevant pivotal tracker stories?
- #163518699
